packages/perseus/src/styles/perseus-renderer.less

```diff
@@ -544,7 +544,7 @@
 
 .perseus-formats-tooltip {
     background: #fff;
-    color: #777;
+    color: #717378;
```

Comment:
Is this a WB color?  Even though we're in a .less file, could you use (or define and use) a variable from `variables.less`?

Reply:
Yes it is. I've moved that color to `variables.less`.